.github/workflows/check_precompiled.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Pass updated files in $updated_files environment variable
+
+# enc_bootloader
+echo "$updated_files" | grep -q "^enc_bootloader/.*\.elf$"
+enc_bootloader_elf_not_updated=$?
+if [ $enc_bootloader_elf_not_updated -eq 1 ]; then
+    echo "Checking enc_bootloader files for modifications as ELFs have not been updated"
+    for file in enc_bootloader/*; do
+        if echo "$file" | grep -q "CMakeLists.txt" || echo "$file" | grep -q "BUILD.bazel"; then
```

Comment:
I _think_ that this could be simplified to
```shell
if [[ "$file" == "enc_bootloader/CMakeLists.txt" ]] || [[ "$file" == "enc_bootloader/BUILD.bazel" ]]; then
```
?